### DISCUSSION TITLE
How do I point kubescape scan at a different image registry to pull the host-scanner image?

### DISCUSSION THREAD

**Discussion by alexdyas** [kubescape/kubescape#1262] [Q&A] [answered]
Hi,

I would like to host the host-scanner container image locally to avoid `kubescape scan` going onto the internet. How do I instruct the kubescape command line util to pull from an alternate source?

Thanks,

Alexander

**Comment by alegrey91** [comment, marked as answer]
Hi @alexdyas, you can use the following command: `kubescape scan --enable-host-scan --host-scan-yaml host-scanner.yaml`.
The flag `--host-scan-yaml` will override the default `DaemonSet` with the file you give as argument.
My advice is to get a copy of this file: https://raw.githubusercontent.com/kubescape/kubescape/master/core/pkg/hostsensorutils/hostsensor.yaml and change the `image: quay.io/kubescape/host-scanner:v1.0.61` according to your local registry location.
P. S. Keep in mind that this functionality will soon be deprecated in the command line.

**Reply by alexdyas** [reply]
Hi @alegrey91,

Thanks for the prompt reply! This sounds like it addresses my issue. However, can you please elaborate on "P. S. Keep in mind that this functionality will soon be deprecated in the command line."? Will there be an alternative way of doing it?

Thanks,

Alexander

**Reply by alegrey91** [reply]
Sure. We decided to remove **host-scanner** from the CLI. This will be available only when running the Kubescape [helm-chart](https://github.com/kubescape/helm-charts/tree/main) operator.
Here's the PR for deprecation that I'm talking about: https://github.com/kubescape/kubescape/pull/1260

**Reply by alexdyas** [reply]
Understood, this sounds reasonable. I will adjust my use accordingly.

Thanks again! Have a great day. (Hi to the cat).
